### PR TITLE
Add representation of disabled state of MaterialTextView.

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: kit/inputs/src/MaterialTextView/MaterialTextView.tsx [hunk 1]
```diff
@@ -97,6 +99,10 @@ function useExtendedProps(
         onFocus: evt => {
             'worklet';
 
+            if (editable.value === false) {
+                return;
+            }
+
```
Comment (savelichalex) [inline kit/inputs/src/MaterialTextView/MaterialTextView.tsx:108]: wait, even though input isn't editable it still fire focus/blur events??? This is weird

**Review by savelichalex — APPROVED** [review]

File: kit/inputs/src/MaterialTextView/MaterialTextView.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply kit/inputs/src/MaterialTextView/MaterialTextView.tsx:108]: This behavior is only on web

Comment (savelichalex) [reply kit/inputs/src/MaterialTextView/MaterialTextView.tsx:108]: Please add a comment, need to think how to avoid it for mobiles 🤔

Comment (SAnatoliiS) [reply kit/inputs/src/MaterialTextView/MaterialTextView.tsx:108]: Or maybe fix this behavior in `useTextViewHandler` for web.

Comment (savelichalex) [reply kit/inputs/src/MaterialTextView/MaterialTextView.tsx:108]: yep, that might work, like for example doesn't return `onFocus` and `onBlur`

**Review by sertony — APPROVED** [review]

**Pull request merged**